starfish/core/spots/DetectSpots/local_max_peak_finder.py

```diff
@@ -70,13 +70,6 @@ def __init__(
         self.is_volume = is_volume
         self.verbose = verbose
 
-        # these parameters are useful for debugging spot-calls
```

Comment:
so. these parameters are indeed useful for debugging spot-calls. is there any way to save them, like for example if you're not using multiprocessing? or is state just impossible to have?

Reply:
debugging is the wrong word. these parameters, per tile, are critical to doing a quality control analysis of the spot calls.

Reply:
but on the other hand, this piece of the code should probably be pulled out of this spot detector and made general for use in all of the other spot detectors. @ambrosejcarr @shanaxel42 and i have talked about this. as such, i suggest we punt this issue over to the spot-finding epic.